ui/receive_page.go

```diff
@@ -0,0 +1,594 @@
+										return common.icons.dropDownIcon.Layout(gtx, values.MarginPadding20)
+									})
+								}),
+							)
+						})
+					}),
+				)
+			})
+		})
+	})
+}
+
+func (wg *walletAccountWidget) walletAccountSection(gtx layout.Context, c pageCommon) layout.Dimensions {
+	sections := func(gtx layout.Context, title string, body layout.Widget) layout.Dimensions {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				txt := c.theme.Body2(title)
+				txt.Color = c.theme.Color.Text
+				inset := layout.Inset{
+					Bottom: values.MarginPadding15,
+				}
+				return inset.Layout(gtx, func(gtx C) D {
+					return txt.Layout(gtx)
+				})
+			}),
+			layout.Rigid(body),
+		)
+	}
+
+	w := []func(gtx C) D{
+		func(gtx C) D {
+			wg.title.Color = c.theme.Color.Text
+			return wg.title.Layout(gtx)
+		},
+		func(gtx C) D {
+			return wg.walletsList.Layout(gtx, len(c.info.Wallets), func(gtx C, i int) D {
+				wn := c.info.Wallets[i].Name
+				accounts := c.info.Wallets[i].Accounts
+				wIndex := i
+
+				inset := layout.Inset{
+					Bottom: values.MarginPadding10,
+				}
+				return inset.Layout(gtx, func(gtx C) D {
+					return sections(gtx, wn, func(gtx C) D {
+						return wg.accountsList.Layout(gtx, len(accounts), func(gtx C, x int) D {
+							accountsName := accounts[x].Name
+							totalBalance := accounts[x].TotalBalance
+							spendable := dcrutil.Amount(accounts[x].SpendableBalance).String()
+
+							click := wg.walletAccounts[wIndex].selectAccount[x]
+							pointer.Rect(image.Rectangle{Max: gtx.Constraints.Max}).Add(gtx.Ops)
+							click.Add(gtx.Ops)
+							wg.walletAccountsHandler(gtx, c, wIndex, x, click)
+
+							return wg.walletAccountsPopupLayout(gtx, accountsName, totalBalance, spendable, c, wIndex, x)
+						})
+					})
+				})
+			})
+		},
+	}
+
+	return wg.walletAccountModal.Layout(gtx, w, 850)
+}
+
+func (wg *walletAccountWidget) Handler(c pageCommon) {
+	for index := 0; index < c.info.LoadedWallets; index++ {
+		if _, ok := wg.walletAccounts[index]; !ok {
+			accounts := c.info.Wallets[index].Accounts
+			if len(accounts) != len(wg.walletAccount.selectAccount) {
+				wg.walletAccount.selectAccount = make([]*gesture.Click, len(accounts))
+				for index := range accounts {
+					wg.walletAccount.selectAccount[index] = &gesture.Click{}
+				}
+			}
+
+			wg.walletAccounts[index] = wg.walletAccount
+		}
+	}
+
+	if wg.fromAccountBtn.Clicked() {
+		wg.isWalletAccountModalOpen = true
+	}
+}
+
+func (wg *walletAccountWidget) walletAccountsHandler(gtx layout.Context, common pageCommon, wIndex, aIndex int, click *gesture.Click) {
+	for _, e := range click.Events(gtx) {
+		if e.Type == gesture.TypeClick {
+			*common.selectedAccount = aIndex
+			*common.selectedWallet = wIndex
+			wg.isWalletAccountModalOpen = false
+		}
+	}
+}
+
+func (wg *walletAccountWidget) walletAccountsPopupLayout(gtx layout.Context, name, totalBal, spendableBal string, common pageCommon, wIndex, aIndex int) layout.Dimensions {
```

Comment:
To many parameters are defined on this method. Try to use very few number of parameters on a method or function to  to keep your code clean and easy to call. I try to limit the number of parameters to 3 or 4 if I can't help it. Anything beyond that, I'll create a data type and pass that to the method. In. You can do the same in this case.